lib/control/v1/authenticate.js

```diff
@@ -1,41 +1,39 @@
 'use strict';
 
+const valid = require('../validation/valid');
+
 const HTTP_OK = 200;
 const HTTP_METHOD_NOT_ALLOWED = 405;
+
 const leaseDuration = 300;
+const bodyParser = require('body-parser');
 
 function Authenticate(app) {
-  const route = app.route('/v1/authenticate');
+  const route = app.use(bodyParser.json()).route('/v1/authenticate');
   const allowedMethods = 'POST';
 
   function methodNotAllowed(req, res) {
     res.set('Allow', allowedMethods);
     res.status(HTTP_METHOD_NOT_ALLOWED);
     res.end();
-  }
+  };
 
   // Express defaults to using the GET route for HEAD requests.
   // So we need to explicitly reject HEAD request.
   route.head(methodNotAllowed);
 
   route.post((req, res, next) => {
-    console.log('FIRST VALIDATION');
-    console.log(req.body);
-    console.log(req.query);
-
-    // Validate that the request body contains two strings: identity document and PKCS7 signature.
-    // if(the.first.test.fails) {
-    //   res.status(403)json(An.error.reponse);
-    //   return;
-    // }
-
+    console.log('authenticate request recieved');
     next();
   },
 
-  // Add in morevalidation as needed
+  function(req, res, next) {
```

Comment:
This doesn't need to be wrapped in a function. Pass the value of `valid.request` directly as a middleware.